src/components/note.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+import './note.scss'
+
+const Note = ({ cls = ``, title = ``, body = `` }) => {
```

Comment:
Why using backquote rather than normal ones (there is no need for variable substitution in these strings)
```suggestion
const Note = ({ cls = "", title = "", body = "" }) => {
```